### DISCUSSION TITLE
A way to choose whether or not you want to use the new 2.8 UI

### DISCUSSION THREAD

**Discussion by malinhiles** [EdgeTX/edgetx#2616] [General]
Lots of great changes on the horizon for 2.8! I'm hoping that there will be some way to enable/disable these UI graphics changes.  Without a way to choose your preferred interface, I think it could be a bit ambitious to implement a new and heavily touch-focused interface that makes the scroll wheel and buttons inconvenient to use.  

Hobbyists will probably not mind the jump, but power users will likely be bummed. As much as I'd like to take advantage of the other new features and improvements, I'd prefer to miss out on future features for the time being so that I can efficiently use the scroll wheel.

The 2.8 UI is a fantastic improvement to the interface for touch screen use and finally makes good use of the touch screen that's been in dormant in some radios for years. But making this drastic change means that the current way most people use this radio will suddenly become pretty inconvenient. I bought my radiomaster tx16s specifically for its modern, well thought-out form,  combined with a solid implementation of the scroll interface. After having negative experiences with both  touch-only radios (14SGA, 16SZ, 32MZ)and button-only radios (OG X9d) in the past, not being able to stick with the current compact and discree UI makes me wary that the scroll based radios will be made pretty inconvenient. 

Let me know your thoughts 🤙

**Comment by offer-shmuely** [comment]
+1 for old/compact UI (non touch optimized) with all the actually good changes!

**Reply by malinhiles** [reply]
Agreed!

**Comment by raphaelcoeffic** [comment]
@malinhiles is this radio a X10? Or what is it actually?

**Reply by malinhiles** [reply]
I am currently using a Radiomaster tx16s

**Comment by offer-shmuely** [comment]
I use both x10s and TX16s
And many of my friends at the club that use TX16s love the openTx/edgeTx, does not care/use the touch at all.
Just like the features

**Comment by jfrickmann** [comment]
When I tested 2.8 rc3 with my EdgeTX touch widgets, the screen was basically frozen. Raphael claims that this is due to the screen having to be rotated 90 degrees on my TX16S, and that takes a toll on graphics performance. So the upgrade that introduces a heavily touch oriented UI apparently also hosed touch on widgets... Go figure!!

**Comment by StefanKellerAC** [comment]
yes, overview is more important (for me), see my comment here:
https://github.com/EdgeTX/edgetx/issues/2062#issuecomment-1291666696

**Reply by malinhiles** [reply]
> yes, overview is more important (for me), see my comment here: [#2062 (comment)](https://github.com/EdgeTX/edgetx/issues/2062#issuecomment-1291666696)

@StefanKellerAC great example. That voices my concern pretty well. Just chimed in over on #2062
